sw/tools/parrot/bebop.py

```diff
@@ -125,8 +125,8 @@ def bebop_status():
     #check firmware version
     v = parrot_utils.check_version(tn, '').strip()
     print("Checking Bebop firmware version... " + v )
-    if v != '3.2.0':
-        print("Error: please upgrade your Bebop firmware to version 3.2.0!")
+    if v < '3.2.0':
```

Comment:
Is this correct to compare the string value? It may be more robust to to convert the string values to a decimal and compare the major, minor and increment values individually.